src/model/paths_types.ml

```diff
@@ -800,6 +818,12 @@ and Resolved_reference : sig
     | `Exception of signature * ExceptionName.t ]
   (** @canonical Odoc_model.Paths.Reference.Resolved.Extension.t *)
 
+  type extension_decl =
+    [ `Identifier of Identifier.reference_extension
+    | `ExtensionDecl of signature * ExtensionName.t
+    | `Exception of signature * ExceptionName.t ]
```

Comment:
Why can `extension_decl` have an `Exception` variant?

Reply:
That's indeed unnecessary, I removed it.